lib/Doctrine/ORM/Mapping/MappingAttribute.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\ORM\Mapping;
+
+/** A marker interface for mapping attributes. */
+interface MappingAttribute extends Annotation
```

Comment:
Why did you name it `MappingAttribute` and not just `Attribute`? It is already inside a `Mapping` namespace.

Reply:
Because literally all classes that implement this interface also have an attribute attached to them named `Attribute` and I didn't want to screw up the imports everywhere. I also wanted the name of the marker interface to be a bit more specific. After all, it's not just an arbitrary attribute, it's one that we use to configure mapping information.